src/core/QCheck2.ml

```diff
@@ -972,11 +985,13 @@ module Observable = struct
 
     let int : int t = (=)
 
+    let bytes : bytes t = (=)
+
     let string : string t = (=)
 
     let bool : bool t = (=)
 
-    let float : float t = (=)
+    let float = (=)
```

Comment:
I think the previous type annotation was there to allow the compiler to specialize the polymorphic equality into a (faster) float equality.

With the float equality fix coming in another PR, I would again prefer to leave this line untouched to keep the diff minimal.

Reply:
Do I remove the type annotation in `QCheck.ml` too?

Reply:
To keep this PR as focused as possible (topic: `bytes` combinators - but also touching `string`), I would prefer "no diff" on any line involving `float`.

That is not to say that a `float` equality fix is not welcome! :pray: It just helps keeping PRs small, managable, and reviewable.
I would expect your forthcoming `float` fix PR to be a two-line diff (one change in QCheck.ml + one in QCheck2.ml) to be accepted and merged immediately.